k8s/BUILD

```diff
@@ -14,3 +14,37 @@
 package(default_visibility = ["//visibility:public"])
 
 licenses(["notice"])  # Apache 2.0
+
+load("@subpar//:subpar.bzl", "par_binary")
+
+py_library(
+    name = "resolver_lib",
+    srcs = ["resolver.py"],
+    deps = [
+        "@containerregistry//:containerregistry",
+        "@yaml//:yaml",
+    ],
+)
+
+py_test(
+    name = "resolver_test",
+    srcs = ["resolver_test.py"],
+    data = [
+        "//examples/hello-grpc/cc/server:server.tar",
+    ],
+    deps = [
+        ":resolver_lib",
+        "@mock//:mock",
+    ],
+)
+
+par_binary(
```

Comment:
It would probably make sense for this to be a standalone tool packaged as part of the containerregistry releases. But this LGTM for now.